Tribler/Core/APIImplementation/LaunchManyCore.py

```diff
@@ -275,9 +278,31 @@ def load_communities():
                                                        load=True,
                                                        kargs=multichain_kwargs)
 
-                        from Tribler.community.tunnel.hidden_community_multichain import HiddenTunnelCommunityMultichain
-                        self.tunnel_community = self.dispersy.define_auto_load(
-                            HiddenTunnelCommunityMultichain, dispersy_member, load=True, kargs=tunnel_kwargs)[0]
+                        if self.session.get_tunnel_community_pooled():
+                            if self.session.get_tunnel_community_test_pooled():
```

Comment:
I don't think we should add test-dependent code to our Core code base. Instead, you can load this test community in the test(s) where you need this.